mmrotate/core/bbox/iou_calculators/rotate_iou2d_calculator.py

```diff
@@ -69,6 +71,14 @@ def rbbox_overlaps(bboxes1, bboxes2, mode='iou', is_aligned=False):
     assert (bboxes1.size(-1) == 5 or bboxes1.size(0) == 0)
     assert (bboxes2.size(-1) == 5 or bboxes2.size(0) == 0)
 
+    # resolve `rbbox_overlaps` abnormal when input rbbox is too small.
```

Comment:
we can simplify the code using the code as below
```
bboxes1 = bboxes1.clamp(min=1e-3)
bboxes2 = bboxes2.clamp(min=1e-3)
```

Reply:
Good suggestion!